apps/prairielearn/src/lib/question-testing.ts

```diff
@@ -181,18 +191,21 @@ async function testVariant(
   course: Course,
   test_type: TestType,
   authn_user_id: string,
+  user_id: string,
```

Comment:
Nit: conventionally `user_id` will come before `authn_user_id` in the list of parameters. Don't forget to update the callsites if you change this.

Reply:
Moved `user_id` before `authn_user_id` in [bc258cb](https://github.com/PrairieLearn/PrairieLearn/pull/10840/commits/bc258cb63105129c9382b9367281d7b040a14698). This was only called once and I updated the order there as well.